transformations/unit_converter/README.md

```diff
@@ -0,0 +1,22 @@
+# Unit converter
+This perturbation adds noise to all types of text sources (sentence, paragraph, etc.).
+
+Author name: Marco Di Giovanni
+Author email: marco.digiovanni@polimi.it
+Author Affiliation: Politecnico di Milano and University of Bologna
+
+## What type of a transformation is this?
+This transformation acts like a perturbation to test robustness.
+It converts length and weight measures to different units (e.g., kilometers to miles) picking at random the new unit but converting accurately the quantity.
+The transformation conserves the format of the original quantity: "100 pounds" is converted to "1600 ounces" but "one-hundred pounds" is converted to "one thousand, six hundred ounces".
+Generated transformations display high similarity to the source sentences.
+
+
+## What tasks does it intend to benefit?
+This perturbation would benefit all tasks which have a sentence/paragraph/document as input like text classification, text generation, etc.
+
```

Comment:
Can you please add a "Data and code provenance" section, which describes where data or code came from, or that it was fully created by you?

Reply:
Added in 5fd898579b6198da3c3a21ece1cc0ca209641b23
The code is fully implemented by me. The table of conversion comes from Wikipedia https://en.wikipedia.org/wiki/Conversion_of_units